packages/fxa-shared/db/models/auth/index.ts

```diff
@@ -80,24 +82,29 @@ export async function getAccountCustomerByUid(uid: string) {
 
 /**
  * Fetch the FxA user id and primary email for a customer by Stripe customer id.
- * @param customerId
- * @returns {Object} result
- * @returns {string} result.uid - The FxA user id
- * @returns {string} result.email - The primary email for the FxA user
  */
 export async function getUidAndEmailByStripeCustomerId(
-  customerId: string
-): Promise<{ uid: string; email: string }> {
+  customerId: string | Stripe.Customer | Stripe.DeletedCustomer | null
+): Promise<{ uid: string | null; email: string | null }> {
```

Comment:
I modified the function signature of `getUidAndEmailByStripeCustomerId` to take a `customer` and do some type checking to make sure it’s a string and throw an error if not.

One possible issue is that I am importing an `error` object from the fxa-auth-server in my new check that `customer` is a string. Is this an issue?